lib/complete.js

```diff
@@ -2,10 +2,14 @@
 import Domodule from 'domodule';
 import Bequest from 'bequest';
 import debounce from 'tinybounce';
+import { on, addClass, removeClass, toArray } from 'domassist';
 
 export default class Complete extends Domodule {
-  get showClass() {
-    return this.options.showClass || 'show';
+  constructor(el) {
+    super(el);
+
+    this.currentIndex = -1;
+    on(this.els.input, 'keyup', event => this.keyup(event));
```

Comment:
Ah, wasn't sure bind was supported well enough. Looks like it is, so I'll switch it.